tests/tests_fabric/parity/test_parity_ddp.py

```diff
@@ -134,6 +134,7 @@ def train_fabric_ddp(fabric):
         pytest.param("cuda", 2, 0.01, marks=RunIf(min_cuda_gpus=2)),
     ],
 )
+@pytest.mark.flaky(reruns=5)
```

Comment:
This might not work well because this is a standalone test, meaning that it needs to run in a separate process

The flaky decorator will re-launch on the same process so this could create undesired interactions.

Reply:
The hope is that the concurrency process ends in the meantime so we can add delay...